examples/bert/user_script.py

```diff
@@ -204,7 +204,7 @@ def eval_accuracy(model: OliveModelHandler, data_dir, batch_size, device, execut
             else:
                 inputs = inputs_i.tolist()
                 input_dict = dict(zip(input_names, [inputs]))
-            res = sess.run(input_feed=input_dict, output_names=None)
+            res = model.run_session(sess, input_feed=input_dict, output_names=None)
```

Comment:
I think this call should now be `model.run_session(sess, input_dict)`?

Reply:
Done!